src/components/cards/style.css

```diff
@@ -0,0 +1,20 @@
+.cardsWrapper {
+    display: flex;
+    flex-direction: column;
+    height: 70vh ;
+    width: 100%;
+    padding: 0 20px 0 20px;
+    overflow-y: auto;
+}
+
+.addCardBlock {
+    display: inline-flex;
+    margin: 13px 10px 0 10px;;
+}
+
+.addCardButton {
+    display: inline-flex;
+    width: 250px;
```

Comment:
The button's width needs to be the same as a card.

Reply:
Done